src/cli.rs

```diff
@@ -622,6 +622,18 @@ pub struct Opts {
     #[arg(long, conflicts_with_all(&["path", "search_path"]), hide_short_help = true, long_help)]
     pub strip_cwd_prefix: bool,
 
+    /// Force including the './' prefix for relative paths. By default, this
+    /// is not included unless using --exec, --exec-batch, or --print0. But with
+    /// this option, the prefix is never stripped.
+    #[arg(
+        long,
+        hide_short_help = true,
+        conflicts_with("absolute_path"),
+        overrides_with("strip_cwd_prefix"),
+        long_help
+    )]
+    pub cwd_prefix: bool,
```

Comment:
Alternatively, I could change the `--strip-cwd-prefix` option to take an enum value that is one of "always", "never", and "auto" (current default behavior). Although in that case I would want to treat it being given without a value as `always`. I'm not sure if clap can handle that.